Angular/angular-mentoring/src/app/directives/border-color.directive.ts

```diff
@@ -0,0 +1,17 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({
+  selector: '[appBorderColorBinding]'
+})
+export class BorderColorDirective {
+  @Input('appBorderColorBinding') public color: string;
+  constructor(private element: ElementRef) { };
+
+  public ngOnInit() {
+    this.highlight(this.color);
+  }
+
+  private highlight(color: string) {
+    this.element.nativeElement.style.border = `5px solid ${color}`;
```

Comment:
Let's use the `Renderer2` to change the style.

For ref: https://angular.io/api/core/Renderer2#setStyle

Reply:
Interesting! Updated.